website/frontend/src/utils/RPCHandler.ts

```diff
@@ -1,6 +1,6 @@
 import io from 'socket.io-client';
```

Comment:
We already removed socket.io in frontend, so this file is unnecessary

Reply:
done, thank you)